prefect_dbt/cloud/utils.py

```diff
@@ -19,3 +22,63 @@ def extract_user_message(ex: HTTPStatusError) -> Optional[str]:
     response_payload = ex.response.json()
     status = response_payload.get("status", {})
     return status.get("user_message")
+
+
+@task(
+    name="Call dbt Cloud administrative API endpoint",
+    description="Calls a dbt Cloud administrative API endpoint",
+    retries=3,
+    retry_delay_seconds=10,
+)
+async def call_dbt_cloud_administrative_api_endpoint(
+    dbt_cloud_credentials: DbtCloudCredentials,
+    path: str,
+    http_method: str,
+    params: Optional[Dict[str, Any]] = None,
+    json: Optional[Dict[str, Any]] = None,
+) -> Any:
+    """
+    Task that calls a specified endpoint in the dbt Cloud administrative API. Use this
+    task if a prebuilt one is not yet available.
+
+    Args:
+        dbt_cloud_credentials: Credentials for authenticating with dbt Cloud.
+        path: The partial path for the request (e.g. /projects/). Will be appended
+            onto the base URL as determined by the client configuration.
+        http_method: HTTP method to call on the endpoint.
+        params: Query parameters to include in the request.
+        json: JSON serializable body to send in the request.
+
+    Returns:
+        The body of the response. If the body is JSON serializable, then the result of
+            `json.loads` with the body as the input will be returned. Otherwise, the
+            body will be returned directly.
+
+    Example:
```

Comment:
@desertaxle can you add an example in the docstring where a user creates a new dbt Cloud job?

80% of the developers will want to do this when using a 3rd party orchestrator with dbt Cloud. 

Example docs JSON request: https://documenter.getpostman.com/view/14183654/UVsSNiXC#09349b37-265e-46ca-a804-c06c43194faf

Reply:
great job on the example LGTM!